docs/pages/pageheader.js

```diff
@@ -570,7 +819,12 @@ function Example() {
           />
         </MainSection.Subsection>
         <MainSection.Subsection
-          description={`PageHeader also supports an optional \`secondaryAction\`. It will likely be a [Button](/button) or an [IconButton](/iconbutton) with a [Tooltip](/tooltip) and optional [Dropdown](/dropdown). Any Buttons or IconButtons should be \`size="lg"\`.`}
+          description={`PageHeader also supports an optional \`secondaryAction\`. It will likely be a [Button](/button) or an [IconButton](/iconbutton) with a [Tooltip](/tooltip) and optional [Dropdown](/dropdown). Any Buttons or IconButtons should be \`size="lg"\`.
+
+Primary and secondary actions are consolidated into [Dropdown](https://gestalt.netlify.app/dropdown) below the [sm breakpoint](https://gestalt.netlify.app/screen_sizes#Web-(px)). \`secondaryAction\` takes both the main component and its equivalent using Dropdown subcomponents.
+
+For example, Button should be complemented with [Dropdown.Item](https://gestalt.pinterest.systems/dropdown#Dropdown.Item), Link should be complemented with [Dropdown.Link](https://gestalt.pinterest.systems/dropdown#Dropdown.Link), and an IconButton displaying a Dropdown should reused the same Dropdown subcomponents. Don't forget to pass \`dropdownAccessibilityLabel\` for the IconButton consolidating all actions into [Dropdown](https://gestalt.netlify.app/dropdown) below the sm breakpoint.
+          `}
```

Comment:
In this example, the second PageHeader gets two "Create new report" dropdown options at small screens
<img width="725" alt="Screen Shot 2022-04-11 at 2 12 48 PM" src="https://user-images.githubusercontent.com/5125094/162834283-c5eb2ad0-94fa-47eb-a022-bc9860a1c898.png">